qiita_pet/support_files/doc/source/dev/rest.rst

```diff
@@ -43,6 +43,8 @@ the important parts of the values are described in the Parameters column.
 |PATCH   | ``/qiita_db/artifacts/<artifact_id>/``                                            | ``op: operation``, ``path: path``,      | Retrieves the artifact information                  | ArtifactHandler            |
 |        |                                                                                   | ``value: value``                        |                                                     |                            |
 +--------+-----------------------------------------------------------------------------------+-----------------------------------------+-----------------------------------------------------+----------------------------+
+|GET     | ``/qiita_db/users/``                                                              |                                         | Retrieves the users emails and names                | UsersListDBHandler         |
```

Comment:
This is a tricky one, I think it should be `Retrieves the email and name of all the users`.